src/librustc_driver/Cargo.toml

```diff
@@ -35,3 +35,4 @@ serialize = { path = "../libserialize" }
 syntax = { path = "../libsyntax" }
 syntax_ext = { path = "../libsyntax_ext" }
 syntax_pos = { path = "../libsyntax_pos" }
+isatty = "0.1"
```

Comment:
Cargo.lock will handle that for you. I don't know if you need this dependency though, I believe you can find equivalent code in rustbuild (search for cfg(Windows)) and possibly in librustc_errors... I'd prefer not to depend on isatty since that means winapi which currently has long build times.